components/formats-gpl/src/loci/formats/in/ND2Handler.java

```diff
@@ -493,7 +493,7 @@ else if (qName.equals("uiMaxDst")) {
           LOGGER.warn("Could not set the pixel type", e);
         }
       }
-      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpc")) {
+      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpcInMemory") || qName.startsWith("uiBpcSignificant")) {
```

Comment:
Do these tags need to be parsed in order of priority, so we pick the most likely to be correct one first?  Is there any potential for non-determinism here in terms of the ordering during parsing which can lead to overwriting the value; should we check the value is set in the conditional to avoid overwriting?

Also, should uiBpcSignificant be handled separately and set bitsPerPixel in the core metadata (or call `setPixelsSignificantBits`) rather than set the pixel type?